account_analytic_account_tag/views/account_analytic_account_views.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 Acsone SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+  <record model="ir.ui.view" id="view_account_analytic_account_form">
+    <field
+            name="name"
+        >account.analytic.account.form (in account_analytic_account_tag)</field>
+    <field name="model">account.analytic.account</field>
+    <field name="inherit_id" ref="analytic.view_account_analytic_account_form" />
+    <field name="arch" type="xml">
+      <field name="plan_id" position="after">
+        <field
+                    name="tag_ids"
+                    widget="many2many_tags"
+                    options="{'color_field': 'color'}"
```

Comment:
IMO it is necessary to apply a domain in order not to link incorrect data as indicated in https://github.com/OCA/account-analytic/pull/572/files#diff-57e0a14fe0a727fc7e8de072048f70a0227ad86ab0662d0422ff73e57ff83fd7R18.

```suggestion
                    options="{'color_field': 'color'}"
                    domain="['|', ('account_analytic_id', '=', False),('account_analytic_id', '=' self)]"
```

Reply:
Hello @victoralmau, 
Although I understand the rationale behind I am unsure that I 100% agree with you as the goal of the new field is to categorize `account.analytic.account` where `account_analytic_id`'s role is to have an impact on how `account.move.line`'s `account.analytic.line`'s tags are set. With that in mind I am unsure that we have to mix both. `account_analytic_id` has an impact on records creation while tag_ids has no effects. With the time passing, and possibly changes on `account_analytic_id` value, that would make the constraint becoming odd I think.
What do you think?

Reply:
I suggested this change to avoid data inconsistencies IMO. 

Currently, if `account_analytic_id` is set on a tag only tags that apply https://github.com/OCA/account-analytic/blob/16.0/account_analytic_tag/models/account_move_line.py#L20C62-L20C62 will be set therefore (even if only informative), allowing to select tags that will never be set if we select that analytic account, I think would cause confusion to the user.

Reply:
I am still confused as when ´account_analytic_id´ will be changed there is no logic to update the ´tag_ids´ field although the records value would not match the new value of ´account_analytic_id´ according to the domain you want to add. I am absolutely not in favor either to update the values, but it was just to highlight the difference between the two fields purposes.

But at the end I'll leave it to you guys. I just wanted to explain what I thought of more precisely here.

Cheers!

Reply:
I think it is not very meaningful this, i.e. `tag_ids` in the analytical account is an informative field, if someone changes something in a tag that affects the analytical account should change it also in the analytical account. Another possibility would be to make the field compute store but maybe that would also be confusing.